cli/src/main/resources/help/credentials.adoc

```diff
@@ -0,0 +1,76 @@
+ CREDENTIALS(1)
+==============
+:doctype: manpage
+
+
+NAME
+----
+credentials - manages keystores that contain ${infinispan.brand.name} Server credentials
+
+
+SYNOPSIS
+--------
+*credentials ls*
+
+*credentials add* 'alias'
+
+*credentials remove* 'alias'
+
+
+DESCRIPTION
+-----------
+List, create, and remove credentials inside a keystore.
+By default, commands manage the `credentials.pfx` keystore in the server configuration directory.
+
+
+SYNOPSIS
+--------
+*credentials ls*::
+Lists credential aliases stored in the keystore.
+
+Add a credential
+
+*credentials add* 'alias'::
+Adds an alias and corresponding credential to the keystore.
+
+Remove a credential
+
+*credentials remove* 'alias'::
+Deletes an alias and corresponding credential from the keystore.
+
+
+OPTIONS
+-------
+*-h, --help*::
+Prints command help.
+
+*-s, --server-root*='path-to-server-root'::
+Specifies the path to the server root directory. Defaults to `server`.
+
+*--path*='credentials.pfx'::
+Specifies the path to the credential keystore. Defaults to the server configuration directory, `server/conf`.
```

Comment:
the `--path` doesn't have a "default". Probably the default should be `credentials.pfx`?